rubyxls.gemspec

```diff
@@ -27,5 +27,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
 
-  spec.add_runtime_dependency "axlsx", "~> 2.0.1"
+  spec.add_runtime_dependency "axlsx", "~> 2.0"
```

Comment:
exactly! The fix is in `axlsx` 2.1.0, and `~> 2.0.1` prevents it from resolving to that version (that can only increment the patch #, not the minor version)